.github/workflows/test-docs.yml

```diff
@@ -0,0 +1,59 @@
+name: Test & Docs
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgis/postgis:12-2.5
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.11"] # ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        dependencies: [normal] # [normal, minimal]
+        database: [postgresql] # maybe check different versions of postgres ?
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+        cache-dependency-path: '**/pyproject.toml'
+    - name: Change dependencies to minimal supported versions
+      run: sed -i -e 's/>=/==/g; s/~=.*==\(.*\)/==\1/g; s/~=/==/g;' pyproject.toml
+      if: matrix.dependencies == 'minimal'
+    - name: Install dependencies
+      run: |
+        sudo apt update
+        sudo apt install libgdal-dev
+        python -m pip install --upgrade pip
+        python -m pip install -e ".[dev,test]"
+        playwright install --with-deps
+    - name: run tests
+      run: pytest .
```

Comment:
I'd use `-vv` to have greater details and the test name in the list ? Maybe even use `make test` (which is `-vx`, but why not) ?

Reply:
Failing fast would help, I think, and verbose means more info. always good. Let's do this :-)